content/200-concepts/100-components/01-prisma-schema/07-referential-actions.mdx

```diff
@@ -0,0 +1,329 @@
+---
+title: 'Referential actions'
+metaTitle: 'Referential actions'
+metaDescription: 'Referential actions let you define the update and delete behavior of related models on the database level'
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+As of [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0] Prisma now supports _referential actions_ as a Preview feature part of the Prisma Schema Language.
+
+This is a [**preview feature**](../../../about/releases#preview) and can be accessed by:
+
+1. [Upgrading to version v2.26.0 of Prisma](../../../guides/upgrade-guides/upgrading-to-latest).
+2. Adding `referentialActions` to the `previewFeatures` field of the Prisma Client [`generator`](../../../reference/api-reference/prisma-schema-reference#generator) block in the `prisma.schema` file.
+
+```prisma file=schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["referentialActions"]
+}
+```
+
+After enabling the preview feature flag you now have 3 options:
```

Comment:
Let's reduce this down to once sentence that links to the upgrade guides - something like:

"When you enable the `referentialActions` preview flag, you **must** choose one of three upgrade paths. See the [2.25.0 upgrade guides](#upgrade-guides) for more information on adopting the feature in your project."

Then move this list to the top of the upgrade page (unless it's already there). It's a very heavy list to have at the top of a conceptual page and I think it's distracting when you're trying to wrap your head around something complex.